### PR TITLE
doc: update documentation for MySQL JDBC release

### PR DESCRIPTION
**Pull request by aaronchung-bitquill**
### Additional Reviewers

@karenc-bq @crystall-bitquill @hsuamz @congoamz @sergiyv-improving @davecramer 

### By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/#semantic-versioning-200).
 
+## [???] - 2023-?-??
```
Comment (crystall-bitquill) [inline CHANGELOG.md:6]: 3.0.0?

File: docs/GettingStarted.md [hunk 2]
```diff
@@ -6,7 +6,9 @@ Before using the AWS Advanced JDBC Driver, you must install:
 
 - Amazon Corretto 8+ or Java 8+.
 - The AWS Advanced JDBC Driver.
-- Your choice of underlying JDBC driver. To use the wrapper with Aurora with PostgreSQL compatibility, you need to install the [PostgreSQL JDBC Driver](https://github.com/pgjdbc/pgjdbc).
+- Your choice of underlying JDBC driver. 
+  - To use the wrapper with Aurora with PostgreSQL compatibility, install the [PostgreSQL JDBC Driver](https://github.com/pgjdbc/pgjdbc).
+  - To use the wrapper with Aurora with MySQL compatibility, install the [MySQL JDBC Driver](https://github.com/mysql/mysql-connector-j).
```
Comment (crystall-bitquill) [inline docs/GettingStarted.md:11]: Does the MariaDB driver need to be added here?

Comment (aaronchung-bitquill) [reply docs/GettingStarted.md]: added!

File: CHANGELOG.md (hunk 1 above)
Comment (aaronchung-bitquill) [reply CHANGELOG.md]: updated!

**Review by karenc-bq — APPROVED** [review]

**Review by karenc-bq — APPROVED** [review]

**Pull request merged**